src/api/app/controllers/webui/cloud/configurations_controller.rb

```diff
@@ -5,6 +5,7 @@ class ConfigurationsController < WebuiController
 
       def index
         @crumb_list.push << 'Configuration'
+        @azure_feature_active = Feature.active?(:cloud_upload_azure)
```

Comment:
:thought_balloon: why don't you use `Feature` in the view?

Reply:
Because views are stupid and should get what comes from the controller and shall rely on anything else. The goal of this approach is to make views reusable.

Reply:
It's the same level of stupidity if you ask a variable `true?` or a class...

Reply:
Could you please explain what speaks against this? For me it sounds like a personal preference. I want to make it consistent and the view reusable when needed and I don't get the point why this should be changed.

Reply:
I have to jump to one more hoop (track down what the `@azure_feature_active` variable means) to understand the view. For no reason.

Reply:
> I have to jump to one more hoop (track down what the @azure_feature_active variable means) to understand the view.

I think `@azure_feature_active` speaks for itself and it's obvious that it contains a `true/false` value.

> For no reason.

It is not for no reason, I just explained you why I added this. I don't like putting logic to the view as it belongs to the controller.

> The view would also not be less reusable because you would always calculate @azure_feature_active with Feature.active?(:cloud_upload_azure) right?

It would, because you're now relying on `Feature.active?` instead of allowing the controller (the controller has the logic part and not the view) to choose what should be done. Fetching that information is not part of a view.

Reply:
I think I have said all there is to say. If you want to ignore my advice I'm fine with it as indicated...

Reply:
@hennevogel I don't ignore anyone, but it can happen that people have different opinions and not always agree. I still think this change is not needed, and therefor I'm asking you if this was meant as a nitpick or a strong request to change this.

Reply:
>  I don't ignore anyone

I don't mean that you are ignoring me as a person. It's about ignoring the arguments I have brought forward. And, as I have written, that's fine for me :nerd_face: 

> it can happen that people have different opinions and not always agree

Not shit Sherlock 🕵🏿! 😋

>  I'd like to ask you if this is a strong request or just a nitpick?

It's [marked as a nitpick](https://github.com/openSUSE/open-build-service/blob/master/CONTRIBUTING.md#how-to-review-code-submissions)...

Reply:
Well, if you all think it's wise to change it, then I do it, but I'm still against this change as it doesn't make sense to me.

Reply:
It does make sense. It is easier to understand and reuse and coherent with the current code. Views shouldn't include logic, as it should be in the controllers... but that is not logic at all.
Please change it.